webui/react/src/services/api.ts

```diff
@@ -1,14 +1,21 @@
 /* eslint-disable @typescript-eslint/camelcase */
+import * as DetSwagger from '@determined-ai/api-ts-sdk';
```

Comment:
non-blocking: Feels weird to call out the technology as part of the variable name but I don't have a better idea so this is probably fine.

Reply:
I agree I'm open to suggestions. DetApi makes the most sense to me but we already have something like that. IMO once we are mostly on the swagger generated  we'd switch it around :thinking:

Reply:
Yup, please don't block on this.